docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -1,185 +1,228 @@
 
-# location of the api-server
-KUBELET_ARGS="--cgroup-driver=systemd --kubeconfig=/etc/kubernetes/master-kubeconfig.yaml --require-kubeconfig"
+    <pre><tt># <b>kubectl get pods --all-namespaces</b>
+    NAMESPACE   NAME                                  READ STATUS RESTARTS AGE
+    kube-system etcd-fedoraatomic27                    1/1 Running  7      56m
+    kube-system kube-apiserver-fedoraatomic2           1/1 Running  8      56m
+    kube-system kube-controller-manager-fedoraatomic27 1/1 Running  0      56m
+    kube-system kube-dns-6f4fd4bdf-krfvt               3/3 Running  0      56m
+    kube-system kube-flannel-ds-dwgw8                  1/1 Running  0       7m
+    kube-system kube-proxy-t9pdb                       1/1 Running  0      56m
+    kube-system kube-scheduler-fedoraatomic27          1/1 Running  0      56m
+    </tt></pre>
 
-# Add your own!
-KUBELET_ARGS=""
+At this point, you have a working kubernetes single-node cluster. Next try running a pod.
 
-```
+## Try Kubernetes with an nginx pod
+To make sure your Kubernetes cluster is working, try running an Nginx (Web server and reverse proxy server) from a pod.
 
-```yaml
-kind: Config
-clusters:
-- name: local
-  cluster:
-    server: http://fed-master:8080
-users:
-- name: kubelet
-contexts:
-- context:
-    cluster: local
-    user: kubelet
-  name: kubelet-context
-current-context: kubelet-context
-```
+1. Get and run three replicas of the nginx pod, opening port 80 to access its content:
 
-* Start the appropriate services on the node (fed-node).
+    <pre><tt># <b>kubectl run nginx --image=nginx --port=80 --replicas=3</b>
+    deployment "nginx" created
+    </tt></pre>
 
-```shell
-for SERVICES in kube-proxy kubelet docker; do 
-    systemctl restart $SERVICES
-    systemctl enable $SERVICES
-    systemctl status $SERVICES 
-done
-```
+2. List information about running pods. The first set of output shows nginx pods being created; the second shows them running:
 
-* Check to make sure now the cluster can see the fed-node on fed-master, and its status changes to _Ready_.
+    <pre><tt># <b>kubectl get pods -o wide</b>
+    NAME                 READY STATUS     RESTARTS   AGE IP     NODE
+    nginx-7587c6fdb6-6mvmt 0/1 ContainerCreating 0   10s <none> fedoraatomic27
+    nginx-7587c6fdb6-6nzq8 0/1 ContainerCreating 0   10s <none> fedoraatomic27
+    nginx-7587c6fdb6-7xssr 0/1 ContainerCreating 0   10s <none> fedoraatomic27
+    # <b>kubectl get pods -o wide</b>
+    NAME                 READY STATUS   RESTARTS AGE IP         NODE
+    nginx-7587c6fdb6-6mvmt 0/1 Running  0        10s 10.244.0.3 fedoraatomic27
+    nginx-7587c6fdb6-6nzq8 0/1 Running  0        10s 10.244.0.4 fedoraatomic27
+    nginx-7587c6fdb6-7xssr 0/1 Running  0        10s 10.244.0.5 fedoraatomic27
+    </tt></pre>
 
-```shell
-kubectl get nodes
-NAME            STATUS      AGE      VERSION
-fed-node        Ready       4h
-```
+3. Expose nginx ports to they are accessible from the host:
```

Comment:
to -> so

Reply:
Good catch again. The word should be "so".

Reply:
Thanks for the feedback @hogepodge . I'll try to make your Cloud Provider Refactoring WG on Wednesday. Is that group limited to cloud providers or is it for operating system providers as well?